docs/index.asciidoc

```diff
@@ -62,6 +62,249 @@ Because the CEF format allows ambiguous timestamp formats, some reasonable assum
    <<plugins-{type}s-{plugin}-default_timezone>>.
  - Localized timestamps are parsed using the provided <<plugins-{type}s-{plugin}-locale>>.
 
+===== Field mapping
+
+The header fields from each CEF payload is expanded to the following fields, depending on whether ECS is enabled:
+
+.Header field mapping
```

Comment:
```suggestion
[id="plugins-{type}s-{plugin}-header-field"]
====== Header field mapping
```

Reply:
Table titles are weird (numbered starting at Table 2, for example). Headings are ranked higher for SEO, and are a better choice.